osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -363,6 +365,13 @@ private void promptDelete()
                 dialogOverlay?.Push(new BeatmapDeleteDialog(Beatmap));
         }
 
+        protected override void Update()
+        {
+            base.Update();
+
+            ensurePlayingSelected();

```

Comment:
isn't this going to do weird things? it means the user can't pause because the track will just instantly start again. it is also calling a potentially expensive `SetExclusiveTrack` function too